.snippets/webservers/Caddyfile-nossl

```diff
@@ -0,0 +1,56 @@
+{
+    servers :80 {
+        timeouts {
+            read_body 120s
+        }
+    }
+}
+
+http://<domain> {
```

Comment:
`http://` is put here, but for the SSL config, there isn't `https://`?

Reply:
`https://` is default for Caddy, therefore is not required. 

If you want to force HTTP, than you have to specify the `http://` prefix.

Reply:
No problem then, but I think `:80` would be better?

Reply:
Personally I prefer `http://`, but I will change it to `:80`, so it is inline with the rest of the documentation. It is OK.